2022-03-25 13:47:54,662 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-25 13:47:54,662 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-25 13:47:54,663 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-25 13:47:54,663 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-25 13:47:54,663 INFO    MainThread:8658 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_134754-lkt6c0hk/logs/debug.log
2022-03-25 13:47:54,663 INFO    MainThread:8658 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_134754-lkt6c0hk/logs/debug-internal.log
2022-03-25 13:47:54,663 INFO    MainThread:8658 [wandb_init.py:init():439] calling init triggers
2022-03-25 13:47:54,663 INFO    MainThread:8658 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-25 13:47:54,663 INFO    MainThread:8658 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 3rwy0v9y
2022-03-25 13:47:54,666 INFO    MainThread:8658 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/3rwy0v9y
2022-03-25 13:47:54,667 INFO    MainThread:8658 [jupyter.py:save_history():429] not saving jupyter history
2022-03-25 13:47:54,667 INFO    MainThread:8658 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-25 13:47:54,667 INFO    MainThread:8658 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-25 13:47:54,669 INFO    MainThread:8658 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-25 13:47:54,683 INFO    MainThread:8658 [wandb_run.py:_restore():1674] restore
2022-03-25 13:47:55,301 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-25 13:47:55,430 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-25 13:47:55,611 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-25 13:47:56,488 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 99430
}

2022-03-25 13:47:56,597 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 99430
}

2022-03-25 13:47:56,699 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}

2022-03-25 13:47:56,807 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}

2022-03-25 13:47:56,915 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}

2022-03-25 13:47:57,022 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}

2022-03-25 13:47:57,141 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}

2022-03-25 13:47:57,247 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}

2022-03-25 13:47:57,356 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}

2022-03-25 13:47:57,468 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}

2022-03-25 13:47:57,842 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}

2022-03-25 13:47:58,414 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 99430
  total_bytes: 99430
}
local_info {
}

2022-03-25 13:47:59,547 INFO    MainThread:8658 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-25 13:47:59,549 INFO    MainThread:8658 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-25 13:47:59,552 INFO    MainThread:8658 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-25 13:47:59,555 INFO    MainThread:8658 [wandb_init.py:init():492] starting backend
2022-03-25 13:47:59,555 INFO    MainThread:8658 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-25 13:47:59,558 INFO    MainThread:8658 [backend.py:ensure_launched():219] starting backend process...
2022-03-25 13:47:59,577 INFO    MainThread:8658 [backend.py:ensure_launched():224] started backend process with pid: 9400
2022-03-25 13:47:59,578 INFO    MainThread:8658 [wandb_init.py:init():501] backend started and connected
2022-03-25 13:47:59,597 INFO    MainThread:8658 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-25 13:47:59,597 INFO    MainThread:8658 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-25 13:47:59,597 INFO    MainThread:8658 [wandb_init.py:init():565] updated telemetry
2022-03-25 13:47:59,636 INFO    MainThread:8658 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-25 13:48:00,482 INFO    MainThread:8658 [wandb_run.py:_on_init():1759] communicating current version
2022-03-25 13:48:00,517 INFO    MainThread:8658 [wandb_run.py:_on_init():1763] got version response 
2022-03-25 13:48:00,517 INFO    MainThread:8658 [wandb_init.py:init():625] starting run threads in backend
2022-03-25 13:48:02,751 INFO    MainThread:8658 [wandb_run.py:_console_start():1733] atexit reg
2022-03-25 13:48:02,751 INFO    MainThread:8658 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-25 13:48:02,752 INFO    MainThread:8658 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-25 13:48:02,752 INFO    MainThread:8658 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-25 13:48:02,752 INFO    MainThread:8658 [wandb_init.py:init():664] run started, returning control to user process
2022-03-25 13:48:02,752 INFO    MainThread:8658 [wandb_watch.py:watch():47] Watching
2022-03-25 13:48:07,812 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set batch_size = 20 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf09bc82e0>>
2022-03-25 13:48:07,812 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb batch_size 20 None
2022-03-25 13:48:07,812 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set epochs = 30 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf09bc82e0>>
2022-03-25 13:48:07,812 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb epochs 30 None
2022-03-25 13:48:07,813 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set lr = 3.0082699717112154e-05 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf09bc82e0>>
2022-03-25 13:48:07,813 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb lr 3.0082699717112154e-05 None
2022-03-25 13:48:07,813 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf09bc82e0>>
2022-03-25 13:48:07,813 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb no_cuda False None
2022-03-25 13:48:07,813 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf09bc82e0>>
2022-03-25 13:48:07,813 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb seed 42 None
2022-03-25 13:48:19,474 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-25 13:48:19,475 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-25 13:48:19,475 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-25 13:48:19,475 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-25 13:48:19,476 INFO    MainThread:8658 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_134819-10i5qu5l/logs/debug.log
2022-03-25 13:48:19,476 INFO    MainThread:8658 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_134819-10i5qu5l/logs/debug-internal.log
2022-03-25 13:48:19,476 INFO    MainThread:8658 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7faedcd0aac0>
2022-03-25 13:48:19,476 INFO    MainThread:8658 [wandb_init.py:init():439] calling init triggers
2022-03-25 13:48:19,476 INFO    MainThread:8658 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-25 13:48:19,476 INFO    MainThread:8658 [wandb_init.py:init():458] re-initializing run, found existing run on stack: lkt6c0hk
2022-03-25 13:48:19,477 INFO    MainThread:8658 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/lkt6c0hk
2022-03-25 13:48:19,477 INFO    MainThread:8658 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-25 13:48:19,478 INFO    MainThread:8658 [wandb_run.py:_restore():1674] restore
2022-03-25 13:48:19,936 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-25 13:48:20,244 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-25 13:48:20,590 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 3530
}

2022-03-25 13:48:20,700 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 54883
}

2022-03-25 13:48:20,814 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 54883
}

2022-03-25 13:48:20,920 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 54883
  total_bytes: 54883
}

2022-03-25 13:48:21,029 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 54883
  total_bytes: 54883
}

2022-03-25 13:48:21,137 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 54883
  total_bytes: 54883
}

2022-03-25 13:48:21,272 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 54883
  total_bytes: 54883
}

2022-03-25 13:48:21,382 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 54883
  total_bytes: 54883
}

2022-03-25 13:48:22,126 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 54883
  total_bytes: 54883
}

2022-03-25 13:48:22,567 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 54883
  total_bytes: 54883
}
local_info {
}

2022-03-25 13:48:23,692 INFO    MainThread:8658 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-25 13:48:23,693 INFO    MainThread:8658 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-25 13:48:23,696 INFO    MainThread:8658 [wandb_run.py:_footer_sync_info():2822] logging synced files
